book/index.md

```diff
@@ -3,23 +3,23 @@ title: Practical Algorithms and Data Structures
 layout: chapter.html
 ---
 
-This book is a practical—and we hope, entertaining—introduction to some
-of the most important data structures and algorithms in computer
+This book is a practical—and, we hope, entertaining—introduction to some
+of the most important algorithms and data structures in computer
```

Comment:
Keep the order of these guys consistent across the book's title, chapter and chapter's content.